examples/trace/main.go

```diff
@@ -37,4 +38,10 @@ func bar(ctx context.Context) {
 	defer span.End()
 
 	// Do bar...
+
+	// Set Datadog APM Trace Metadata
+	span.AddAttributes(
+		trace.StringAttribute(ext.ResourceName, "my-app-resource"),
```

Comment:
This is a bit confusing because the resource is already defaulting to `/bar` in line 36, which is more correct than `my-app-resource`. Maybe we can use the value `/foo/bar` which could be ok, since `bar` is a child of `foo`, and we'd also illustrate that the resource can be overridden?

Reply:
Ah done